lol.janjic.Splitcat.yml

```diff
@@ -0,0 +1,48 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: lol.janjic.Splitcat
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: splitcat
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: Splitcat
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - |
+        mv splitcat /app/
+        chmod +x /app/splitcat/splitcat
+        mkdir -p /app/bin
+        ln -s /app/splitcat/splitcat /app/bin/splitcat
+
+        echo "$FLATPAK_DEST"
+
+        ls /app/splitcat
+
+        install -Dm644 /app/splitcat/$FLATPAK_ID.png -t $FLATPAK_DEST/share/icons/hicolor/256x256/apps/
+        install -Dm644 /app/splitcat/$FLATPAK_ID.desktop -t $FLATPAK_DEST/share/applications/
+        install -Dm644 /app/splitcat/$FLATPAK_ID.appdata.xml -t $FLATPAK_DEST/share/metainfo
+    sources:
+      - type: archive
+        url: https://github.com/vogonwann/splitcat/releases/download/v0.1.7/splitcat-linux-x64.tar.gz
+        dest: splitcat
+        strip-components: 0
+        sha256: def3ac3deaa6925d0335e4b5f33260bfe4a87ef2d0e7f0626836c0ef5aa098bc
+        only-arches:
+          - x86_64
+      - type: archive
+        url: https://github.com/vogonwann/splitcat/releases/download/v0.1.7/splitcat-linux-arm64.tar.gz
```

Comment:
fix the icon name here, it's called icon.png for aarch64 and lol.janjic.Splitcat.png for x86_64.

Your build commands are common for both.

Reply:
fixed